### DISCUSSION TITLE
"Admin Password Reset" feature

### DISCUSSION THREAD

**Discussion by yurividal** [dani-garcia/vaultwarden#1891] [Ideas]
Hi. Is [this](https://bitwarden.com/help/article/admin-reset/)  feature available in vaultwarden? If so, is how to enable it? If not, is it planned to be added?

**Comment by meichthys** [comment]
It is not currently available, but there has been talk about adding the necessary code to make it work. I sure hope it's implemented soon. It's a necessary feature for organizations to prevent users from being locked out if they don't properly setup recovery keys, etc.

**Comment by meichthys** [comment]
We should probably close this discussion in favor of #1820

**Comment by meichthys** [comment]
@BlackDex,
Can we close this discussion in favor of https://github.com/dani-garcia/vaultwarden/discussions/1820 ?

**Reply by BlackDex** [reply]
Discussions can't be closed. I can only Lock it, but i think two notifications pointing to the other discussion is good enough for me ;)

**Reply by meichthys** [reply]
Oh, ok. Thanks.
Looking forward to using vaultwarden once this feature is implemented!
